### DISCUSSION TITLE
How can I seek the Information about patients' gestational age

### DISCUSSION THREAD

**Discussion by z-s-p** [MIT-LCP/mimic-code#1554] [MIMIC-IV]
Dear MIMIC-IV Community Members,
   I am conducting a study related to pregnancy women (maternal near-miss). I want to know the gestational age of pregnancy women, but there are no specific code for gestational age  in ICD-9 codes.  How can I seek the Information about patients' gestational age?
   Thanks in advance!

**Comment by alistairewj** [comment]
Discharge summary tends to have that info (e.g. 39w0d).
